docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
+└──────────────────────────────────────────────────────────────────────────────┘
+• Checked output: stdout
+• Missing file containing the expected output: tests/snapshots/my_project/5d41402abc4b/stdout
+• Path to captured stdout: _build/testo/status/my_project/5d41402abc4b/stdout
+• Path to captured log: _build/testo/status/my_project/5d41402abc4b/log
+• Log (stderr) is empty.
+────────────────────────────────────────────────────────────────────────────────
+1/1 selected test:
+  1 successful (1 pass, 0 xfail)
+  0 unsuccessful (0 fail, 0 xpass)
+1 test whose output needs first-time approval
+overall status: failure</code></pre>
+<p>This was expected since we don’t have a reference output for our test. First, we’re going to check that the captured output is what we were expecting:</p>
+<pre><code>$ cat _build/testo/status/my_project/5d41402abc4b/stdout
+hello!</code></pre>
+<p>Note that at any time, we can get a summary of the tests that need attention using <code>./test status</code>, without having to re-run the tests:</p>
+<pre><code>$ ./test status
+[PASS*] 5d41402abc4b hello</code></pre>
+<p>We can see details with the <code>-l</code> (“long output”) option:</p>
+<pre><code>┌──────────────────────────────────────────────────────────────────────────────┐
+│ [PASS*] 5d41402abc4b hello                                                   │
+└──────────────────────────────────────────────────────────────────────────────┘
+• Checked output: stdout
+• Missing file containing the expected output: tests/snapshots/my_project/5d41402abc4b/stdout
+• Path to captured stdout: _build/testo/status/my_project/5d41402abc4b/stdout
+• Path to captured log: _build/testo/status/my_project/5d41402abc4b/log
+• Log (stderr) is empty.
+────────────────────────────────────────────────────────────────────────────────
+1/1 selected test:
+  1 successful (1 pass, 0 xfail)
+  0 unsuccessful (0 fail, 0 xpass)
+1 test whose output needs first-time approval
+overall status: failure</code></pre>
+<p>Then, we’re going to approve this output and make it the reference snapshot with <code>./test approve</code>:</p>
+<pre><code>$ ./test approve
+Expected output changed for 1 test.</code></pre>
+<p>Let’s check the new status:</p>
+<pre><code>$ ./test status</code></pre>
+<p>Nothing is printed because all the tests passed and are in the best state possible. To see the full list of tests, use <code>-a</code> (“all”):</p>
+<pre><code>$ ./test status -a
+[PASS]  5d41402abc4b hello</code></pre>
+<p>Now, there should be a snapshot file somewhere in our file system. Git shows us that <code>tests/snapshots</code> was created:</p>
+<pre><code>$ git status
+...
+Untracked files:
+  (use &quot;git add &lt;file&gt;...&quot; to include in what will be committed)
+    tests/snapshots/</code></pre>
+<p>The snapshot files are organized as follows:</p>
+<pre><code>$ tree tests/snapshots/
+tests/snapshots/
+└── my_project
+    └── 5d41402abc4b
```

Comment:
maybe explain quickly why you have this magic number here, can say derived from
test name but digest to avoid issue with special chars in the test name string?

Reply:
and maybe link to ref manual saying you can choose the name of the snapshot?